bukkit/src/main/java/dev/aurelium/auraskills/bukkit/requirement/BukkitLootRequirements.java

```diff
@@ -0,0 +1,31 @@
+package dev.aurelium.auraskills.bukkit.requirement;
+
+import dev.aurelium.auraskills.api.loot.LootRequirements;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+
+import java.util.List;
+import java.util.UUID;
+
+public class BukkitLootRequirements extends LootRequirements {
+
+    private final List<RequirementNode> nodes;
+
+    public BukkitLootRequirements(List<RequirementNode> nodes) {
+        this.nodes = nodes;
+    }
+
+    public boolean checkByUuid(UUID uuid) {
+        return check(Bukkit.getPlayer(uuid));
```

Comment:
Can you add a null check on the value of getPlayer and return false if it's null? Extract Player to a local variable to do this.